src/ci/stage-build.py

```diff
@@ -851,6 +856,13 @@ def run(runner: BenchmarkRunner):
 
     build_args = sys.argv[1:]
 
+    # Skip components that are not needed for try builds to speed them up
+    if is_try_build():
+        LOGGER.info("Skipping building of unimportant components for a try build")
+        for target in ("rust-docs", "rustc-docs", "rust-docs-json", "rust-analyzer", "clippy",
```

Comment:
hmm, should this also exclude miri and rustfmt?

really what you want is "build.extended=false and also cargo" - i don't remember if `extended` is a hard off switch or a default today. but you could pass it with `--set build.extended=false` on the command line since recently.

Reply:
Sadly it looks like the perf bot requires the extended step artifacts. It fails to start on this PR when the extended archive isn't built.